12.11_예산.js

```diff
@@ -0,0 +1,15 @@
+function solution(d, budget) {
+    d.sort(function(a,b){
+        return a - b
+    })
+    let anwser = 0
+    for (let num1 = 0; num1 <= 100; num1++){
+        budget -= d[num1]
+        if (budget >= 0){
+            anwser++
+        }
+        else
+        break;
```

Comment:
```suggestion
        budget >= 0 ? anwser++ : break;
```
```anwser```철자 오류 ```answer```  삼항연산자로도 가능할 것 같습니다.

[참고사이트](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Conditional_operator)